bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sncf"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="station">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api"/>
+		</supported-bridge-type-refs>
+
+		<label>Station</label>
+
+		<channel-groups>
+			<channel-group id="arrivals" typeId="passage">
+				<label>Next Arrival</label>
+			</channel-group>
+			<channel-group id="departures" typeId="passage">
+				<label>Next Departure</label>
+			</channel-group>
+		</channel-groups>
+
+		<representation-property>stopPointId</representation-property>
+
+		<config-description>
+			<parameter name="stopPointId" type="text" required="true">
+				<label>Station ID</label>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="passage">
+		<label>Other</label>
+		<channels>
+			<channel id="direction" typeId="direction"/>
+			<channel id="code" typeId="code"/>
+			<channel id="commercialMode" typeId="commercialMode"/>
```

Comment:
I am not sure the user needs two channels. What he needs is the "line". As an example, having "RER" as commercial mode + "A" as code in two different channels could be replaced by a simple channel that will deliver "RER A".
WDYT ?

Reply:
I will check if it works also for buses, but basically, you're right.

Reply:
Changed.